app/app - db_Estimate.py

```diff
@@ -47,10 +49,5 @@
                     AND cpc.county = %s
                     AND CAST(date_part('year', cpc.date_code) AS varchar) = '2018';
             """ {comp_states[    query_state[1], query_state[0]}
-# set up loop to find the most accurate estimate (relaxing pop)
-
-# set up loop to find the most accurate estimate (relaxing comps)
-
 
-con.close
-    
+con.close
```

Comment:
You may want to look into linters - [pylint](https://www.pylint.org/) or [black](https://github.com/psf/black) are popular python tools that will deal with this newline issue automatically.

Reply:
👍 Added pylint issue